Nemt/nemt.cpp

```diff
@@ -8,19 +8,27 @@ Nemt::Nemt(int cl) {
     counter = 0;
 }
 
+inline int Nemt::clear_return(int return_value) {
```

Comment:
`clear_return` doesn't exactly say what this function does. Can we find a better name?